### PR TITLE
Organize snapshots assembly, update block PVs

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.14

Issue: None

Link to docs preview: 
- [Creating block PV](https://63512--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-block#virt-creating-local-block-pv_virt-uploading-local-disk-images-block)
- [Managing snapshots](https://63512--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots)

QE review: to be reviewed by Kevin Alon Goldblatt
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc [hunk 1]
```diff
@@ -18,36 +18,40 @@ Online snapshots have a default time deadline of five minutes (`5m`) that can be
...
 
-include::modules/virt-updating-virtio-drivers-windows.adoc[leveloffset=+2]
+include::modules/virt-verifying-online-snapshot-creation-with-snapshot-indications.adoc[leveloffset=+1]
 
-include::modules/virt-creating-vm-snapshot-web.adoc[leveloffset=+1]
+[id="restoring-vms-from-snapshots"]
+== Restoring virtual machines from snapshots
 
-include::modules/virt-creating-vm-snapshot-cli.adoc[leveloffset=+1]
+You can restore virtual machines (VM) from snapshots by using the {product-title} web console or the command line.
```
Comment (kgoldbla) [inline virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc:41]: Here you use:
 virtual machines (VM) other places
 virtual machines (VMs)

File: modules/virt-creating-local-block-pv.adoc [hunk 2]
```diff
@@ -18,25 +17,24 @@ virtual machine image.
...
+# dd if=/dev/zero of=loop10 bs=100M count=20
 ----
 
-. Mount the `loop10` file as a loop device.
+. Mount the `loop10` file as a loop device:
 +
 [source,terminal]
 ----
-$ losetup </dev/loop10>d3 <loop10> <1> <2>
+# losetup /dev/loop10 loop10 <1>
```
Comment (kgoldbla) [inline modules/virt-creating-local-block-pv.adoc:35]: Do you mean that the path and the file name are now one parameter where previously it was two?

File: modules/virt-configuring-secondary-network-vm-live-migration.adoc [hunk 3]
```diff
@@ -6,57 +6,55 @@
...
-To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition for the `openshift-cnv` namespace by using the CLI. Then, add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
+To configure a dedicated secondary network for live migration, you must first create a bridge network attachment definition (NAD) by using the CLI. Then, you add the name of the `NetworkAttachmentDefinition` object to the `HyperConverged` custom resource (CR).
 
 .Prerequisites
 
 * You installed the OpenShift CLI (`oc`).
 * You logged in to the cluster as a user with the `cluster-admin` role.
-* The Multus Container Network Interface (CNI) plugin is installed on the cluster.
-* Every node on the cluster has at least two Network Interface Cards (NICs), and the NICs to be used for live migration are connected to the same VLAN.
-* The virtual machine (VM) is running with the `LiveMigrate` eviction strategy.
```
Comment (kgoldbla) [inline modules/virt-configuring-secondary-network-vm-live-migration.adoc:17]: Not sure about this one - Is the eviction strategy no longer a prerequisite

File: modules/virt-creating-local-block-pv.adoc (hunk 2 above)
Comment (apinnick) [reply modules/virt-creating-local-block-pv.adoc:35]: No, they are still 2 parameters, but with 1 callout explaining what they are. Multiple callouts on the same line of code look messy.

Comment (apinnick) [reply modules/virt-creating-local-block-pv.adoc:35]: Also "d3" appears to have been an error that was introduced a while ago and never caught.

File: modules/virt-configuring-secondary-network-vm-live-migration.adoc (hunk 3 above)
Comment (apinnick) [reply modules/virt-configuring-secondary-network-vm-live-migration.adoc:17]: Eviction strategy is not a prerequisite for creating a migration network. It's a prerequisite for migrating the VM. That's why I'm removing it from this procedure.

File: modules/virt-configuring-secondary-network-vm-live-migration.adoc
Comment (jeana-redhat) [inline modules/virt-configuring-secondary-network-vm-live-migration.adoc:1]: I don't see this topic or its assembly in the preview - is it obsolete or am I just looking in the wrong place? Edits look fine in this view in any case.

File: modules/virt-creating-vm-snapshot-cli.adoc [hunk 4]
```diff
@@ -23,40 +16,38 @@ The QEMU guest agent takes a consistent snapshot by attempting to quiesce the VM
...
-of the `VirtualMachineSnapshot` object.
+. Create the `VirtualMachineSnapshot` object:
 +
 [source,terminal]
 ----
-$ oc create -f <my-vmsnapshot>.yaml
+$ oc create -f <snapshot_name>.yaml
 ----
++
+The snapshot controller creates a `VirtualMachineSnapshotContent` object, binds it to the `VirtualMachineSnapshot` and updates the `status` and `readyToUse` fields of the `VirtualMachineSnapshot` object.
```
Comment (jeana-redhat) [inline modules/virt-creating-vm-snapshot-cli.adoc:41]:
```suggestion
The snapshot controller creates a `VirtualMachineSnapshotContent` object, binds it to the `VirtualMachineSnapshot`, and updates the `status` and `readyToUse` fields of the `VirtualMachineSnapshot` object.
```

**Review by jeana-redhat — COMMENTED** [review]
Excellent enhancements :)

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**